test/integration/scheduler/scheduler_test.go

```diff
@@ -305,6 +305,53 @@ var _ = ginkgo.Describe("Scheduler", func() {
 			})
 		})
 
+		ginkgo.It("Should admit workloads when resources are dynamically reclaimed", func() {
+			firstWl := testing.MakeWorkload("first-wl", ns.Name).Queue(prodQueue.Name).
+				PodSets(
+					*testing.MakePodSet("first", 1).Request(corev1.ResourceCPU, "3").Obj(),
+					*testing.MakePodSet("second", 1).Request(corev1.ResourceCPU, "1").Obj(),
+					*testing.MakePodSet("third", 1).Request(corev1.ResourceCPU, "1").Obj(),
+				).
+				Obj()
+			ginkgo.By("Creating first workload", func() {
+				gomega.Expect(k8sClient.Create(ctx, firstWl)).Should(gomega.Succeed())
+
+				util.ExpectWorkloadsToHaveQuotaReservation(ctx, k8sClient, prodClusterQ.Name, firstWl)
```

Comment:
Let's use `ExpectWorkloadsToBeAdmitted`.

Reply:
IIUC it was renamed to `ExpectWorkloadsToHaveQuotaReservation` here: https://github.com/kubernetes-sigs/kueue/commit/16e21c6063a846b617e77fa32ecf2c5917489af1

Reply:
Right, we should add back a separate `ExpectWorkloadsToBeAdmitted`.

That's what we care about in all these tests. The `QuotaReserved` is just an intermediate step that might or might not match Admitted status, depending on the configuration.

Reply:
Okey, I will add it

Reply:
added https://github.com/kubernetes-sigs/kueue/pull/1241/commits/ea67cf0003fe2de21d7c90d6613c6314490af050